.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Bug: '
+labels: ''
+assignees: ''
+
+---
+
+<!--- Please provide a short description of the bug -->
+<!--- Please follow the naming conventions -->
+
+### Description
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+### Expected Behavior
+<!--- Tell us what should happen -->
+
+### Actual Behavior
+<!--- Tell us what happens instead -->
+
+### Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+### Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+
+### Context
+<!--- How has this bug affected you? What were you trying to accomplish? -->
+
+### Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used:
```

Comment:
Info about how to get the actual version  might be useful too.

Reply:
Want to keep this generic and push to every project so that we can sync this.